apps/bsatool/CMakeLists.txt

```diff
@@ -18,3 +18,7 @@ if (BUILD_WITH_CODE_COVERAGE)
   add_definitions (--coverage)
   target_link_libraries(bsatool gcov)
 endif()
+
+if(MSVC)
+  target_link_libraries(bsatool imm32.lib winmm.lib version.lib)
```

Comment:
The targets using SDL would be launcher, openmw and components.

Then it should be handled in the SDL2 cmake script. Actually I see this in FindSDL2.cmake:
<pre>

if(WIN32)
    list(APPEND SDL2_TARGET_SPECIFIC winmm imm32 version msimg32)
endif()

set(SDL2_PROCESS_LIBS SDL2_TARGET_SPECIFIC)

libfind_process(SDL2)
</pre>

Then, somewhere inside the libfind_process macro it's doing something with the SDL2_PROCESS_LIBS.

Clearly, the find script already attempts to handle the libraries you are missing. We should look into why that is not working.

Does reverting this PR help? https://github.com/OpenMW/openmw/pull/1027